trade-finance-manager-ui/server/constants/primary-navigation-items.ts

```diff
@@ -0,0 +1,5 @@
+export const PRIMARY_NAVIGATION_ITEMS = {
```

Comment:
I'd lose `_ITEMS` from the name as it doesn't really add anything. Maybe just `PRIMARY_NAVAGATION` or `PRIMARY_NAVIGATION_KEY`? Not a blocker though if you'd prefer to keep as is.